cvmfs/options.cc

```diff
@@ -311,14 +320,34 @@ void OptionsManager::PopulateParameter(
              param.c_str(), iter->second.c_str(), val.value.c_str());
     return;
   }
-
+  ParseValue(&val, opt_templ_mgr);
   config_[param] = val;
   if (taint_environment_) {
     int retval = setenv(param.c_str(), val.value.c_str(), 1);
     assert(retval == 0);
   }
 }
 
+void OptionsManager::ParseValue(ConfigValue *val,
+  OptionsTemplatingManager *opt_templ_mgr) {
+  if (opt_templ_mgr == NULL) {
+    // No template manager -> nothing to do here
+    return;
+  }
+  vector<string> tokens = SplitString(val->value, '@');
+  if (tokens.size() > 2) {
+    string resStr = "";
+    for (unsigned int i = 0; i < tokens.size(); i++) {
+      if (i%2 == 0) {
+        resStr+=tokens[i];
+      } else {
+        resStr+=opt_templ_mgr->GetVal(tokens[i]);
```

Comment:
white spaces around operator

Reply:
I think we should through a `LogCvmfs` statement, at least at the debug level, if there is an unrecognized template identifier.